tests/integration/functional/cli/test_model.py

```diff
@@ -0,0 +1,326 @@
+        name="arias_registry",
+        id=uuid4(),
+        flavor="test",
+        config={},
+        type=StackComponentType.MODEL_REGISTRY,
+        user=uuid4(),
+        workspace=uuid4(),
+        created=datetime.now(),
+        updated=datetime.now(),
+    )
+
+
+def test_list_models(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_registered_models = (
+        cli.commands["model-registry"].commands["models"].commands["list"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "list_models",
+        return_value=concrete_registered_models,
+    )
+    # Run the command
+    result = runner.invoke(list_registered_models, obj=concrete_model_registry)
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_update_model(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    update_registered_model = (
+        cli.commands["model-registry"].commands["models"].commands["update"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "update_model",
+        return_value=concrete_registered_models[0],
+    )
+    # Run the command
+    result = runner.invoke(
+        update_registered_model,
+        ["test_model", "--description", "new_test_description"],
+        obj=concrete_model_registry,
+    )
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_get_model(
+    mocker,
+    concrete_registered_models,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_registered_model = (
+        cli.commands["model-registry"].commands["models"].commands["get"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "get_model",
+        return_value=concrete_registered_models[0],
+    )
+    # Run the command
+    result = runner.invoke(
+        list_registered_model, ["test_model"], obj=concrete_model_registry
+    )
+
+    # Check the result
+    assert result.exit_code == 0
+
+
+def test_list_model_versions(
+    mocker,
+    concrete_registered_model_version,
+    concrete_model_registry,
+):
+    # Get the list command
+    runner = CliRunner()
+    list_model_versions = (
+        cli.commands["model-registry"]
+        .commands["models"]
+        .commands["list-versions"]
+    )
+
+    # Patch the find_model_server method
+    mocker.patch.object(
+        ConcreteModelRegistry,
+        "list_model_versions",
+        return_value=concrete_registered_model_version,
+    )
+    # Run the command
+    result = runner.invoke(
+        list_model_versions, ["test_model"], obj=concrete_model_registry
+    )
+
+    # Check the result
+    assert result.exit_code == 0
```

Comment:
Same here. Confirm listing model versions when there aren't any. Also for this test, check the `result.output` to ensure that our model version is mentioned.